tensorflow_datasets/core/dataset_info.py

```diff
@@ -447,19 +448,19 @@ def show_examples(self, ds, rows=3, cols=3, plot_scale=3):
     image_key = image_keys[0]
     num_examples = rows * cols
     examples = list(dataset_utils.as_numpy(ds.take(num_examples)))
-    fig = plt.figure(figsize=(plot_scale*cols, plot_scale*rows))
+    fig = tfds.core.lazy_imports.matplotlib.pyplot.figure(figsize=(plot_scale*cols, plot_scale*rows))
```

Comment:
It is good to avoid reusing code whenever possible.
Here, it would be cleaner to have:
```
plt = tfds.core.lazy_imports.matplotlib.pyplot

plt.imshow(...)
plt.xlabel(...)
```